src/version/get_server_version/v1.rs

```diff
@@ -33,7 +37,51 @@ impl Request {
 
 impl Response {
     /// Creates a `Response` with the given Synapse and Python versions.
-    pub fn new(server_version: String, python_version: String) -> Self {
+    pub fn new(server_version: String, python_version: Option<String>) -> Self {
```

Comment:
Could you just remove the `python_version` parameter instead? I don't think a Rust server implementation usually has a reason to set this field, and it can still be set to `Some` via direct field access after creating the response with `Response::new`.

It's a breaking change either way.

Reply:
Do you mean something like this?
```rust
    pub fn new(server_version: String) -> Self {
        Self { server_version, python_version: None }
    }
```
Or completely removing the python_version from the Response struct (which would break old versions no?)

Reply:
Yes, I mean exactly what you put in the code snippet. Removing the fields wouldn't break clients that deserialize this response and only check the synapse version field, but I think it's slightly more useful to keep the field, but remove it from the constructor.

Reply:
Done ✅ Thank you so much for your review!